LayoutTests/fast/canvas/canvas-size-minimum-limit.html

```diff
@@ -0,0 +1,23 @@
+<head>
+    <script src="../../resources/js-test-pre.js"></script>
```

Comment:
Please use `js-test.js`. Your test output is incomplete at the moment because you're using pre.js without post.js.